### ISSUE TITLE
cChocoSource doesn't update priority

### ISSUE THREAD

**Issue by bgelens** [PowerShellOrg/cChoco#63]
If a source is already registered, the priority won't be adjusted when changed in the configuration.

Repo:
```powershell
cChocoSource MySource {
    Name = 'MySource'
    Ensure = 'Present'
    source = 'http://localhost:8080/chocolatey/'
    Priority = 10
}
```
Then try to adjust (won't happen):
```powershell
cChocoSource MySource {
    Name = 'MySource'
    Ensure = 'Present'
    source = 'http://localhost:8080/chocolatey/'
    Priority = 0
}
```

**Comment by ferventcoder** [comment]
This is now fixed with your test?

**Comment by bgelens** [comment]
yes, fixed with pr #64 
Test-TargetResource wasn't evaluating the priority. Now when actual is different from desired, Set is called and choco is smart enough to overwrite / update
